src/coreclr/jit/target.h

```diff
@@ -170,6 +177,27 @@ enum _regMask_enum : unsigned
 #include "register.h"
 };
 
+#elif defined(TARGET_LOONGARCH64)
```

Comment:
This is identical to the `TARGET_ARM` case (but not the arm64 case): just add `|| defined(...)` there.

Reply:
OK, Thanks